### ISSUE TITLE
Merging output into single file

### ISSUE THREAD

**Issue by herkyl** [tarruda/grunt-coffee-build#1]
How can I merge all files into one single file with mappings back to original coffee files?
I don't understand if this is possible, if it is then modifying documentation would be nice.

**Comment by tarruda** [comment]
Yes its possible, just specify a 'dest' that does END(*update) with '.js'. The main page readme shows all possible examples. I have also written a grunt-init template that configures everything: https://github.com/tarruda/grunt-init-umd-commonjs-coffee. I'm going to update the documentation soon to specify that more explicitly.
